### PR TITLE
Added Best/First/Worst Fit algorithm implementation

### PR DESCRIPTION
**Pull request by DekasDimitrios**
@stepfencurryxiao Here is my pull request. Could you please review it? :) Thanks.

### CODE REVIEW

File: Others/BestFit.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
+package Others;
+
+import java.util.Array-List;
```
Comment (stepfencurryxiao) [inline Others/BestFit.java]: import java.util.ArrayList;

**Comment by DekasDimitrios** [comment]
@stepfencurryxiao Probably a typo. Change done.

**Comment by stepfencurryxiao** [comment]
Thank you! Merged.

**Comment by DekasDimitrios** [comment]
Thank you for the time and effort! Glad you approved it! :)

**Comment by stepfencurryxiao** [comment]
You're welcome:-)

**Pull request merged**